Platform/ARM/N1Sdp/N1SdpPlatform.dsc

```diff
@@ -4,7 +4,8 @@
 # This provides platform specific component descriptions and libraries that
 # conform to EFI/Framework standards.
 #
-# Copyright (c) 2018 - 2024, ARM Limited. All rights reserved.<BR>

+# Copyright (c) 2018 - 2024, ARM Limited. All rights reserved.<BR>

```

Comment:
This hunk suggests the modifications may be using the wrong line endings.

Reply:
I will double check, it's possible my edk2-platforms has a messed up configuration. It also may be this file had a couple weird line endings, the ^M that was currently checked into the file makes it look like someone applied a patch via copy/paste and checked it in.

Reply:
Line endings look correct on my side. I think the existing line endings in this file were weird from the copy/paste, but I will edit this to leave those lines alone.